shell/platform/android/android_image_generator.cc

```diff
@@ -191,7 +200,7 @@ void AndroidImageGenerator::NativeImageHeaderCallback(JNIEnv* env,
                                                       int width,
                                                       int height) {
   AndroidImageGenerator* generator =
-      reinterpret_cast<AndroidImageGenerator*>(generator_address);
+      Jlong2Pointer<AndroidImageGenerator*>(generator_address);
```

Comment:
This change is not needed.  The original `reinterpret_cast<AndroidImageGenerator*>` should be safe here.

Reply:
Done.